### PR TITLE
chore(android): Update splash screen with SIL Tai Heritage Pro logo

### PR DESCRIPTION
**Pull request by darcywong00**
Addresses #12053 for Android

Updates splash screen with [SIL Tai Heritage Pro](https://sites.google.com/sil.org/brand-resource-center/branding/visual-brand/logos#h.jg2j17pi3kaf) logo along with
> When we do so, let's make it a lot smaller than it currently is

### Screens for comparison

| Current | With the change |
|-------------|-------------------------|
|splash-master | splash-tai-heritage |

@keymanapp-test-bot skip

### CODE REVIEW

File: android/KMAPro/kMAPro/src/main/res/drawable-hdpi/sil_logo_color_tai_heritage_pro.9.png
Comment (mcdurdin) [inline android/KMAPro/kMAPro/src/main/res/drawable-hdpi/sil_logo_color_tai_heritage_pro.9.png:1]: What's with the borders on the image?

**Review by mcdurdin — APPROVED** [review]
LGTM, just puzzled about the borders

File: android/KMAPro/kMAPro/src/main/res/drawable-hdpi/sil_logo_color_tai_heritage_pro.9.png
Comment (darcywong00) [reply android/KMAPro/kMAPro/src/main/res/drawable-hdpi/sil_logo_color_tai_heritage_pro.9.png:1]: Hmm, maybe some artifact introduced by the tool I used to generate the 9-patch file. Manually removed...

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.86-alpha](https://keyman.com/downloads/releases/alpha/18.0.86)

**Pull request merged**